### ISSUE TITLE
No response for vulnerability report 

### ISSUE THREAD

**Issue by asi-cider** [HangfireIO/Hangfire#2242]
Hey,

I've reported a serious vulnerability 2 weeks ago to security@hangfire.io and got no response. Is there a different way to report  vulnerabilities?

**Comment by odinserj** [comment]
Hi @asi-cider, I was thinking it's some kind of scam as the alternative repository doesn't look reputable for me (I had some issues with vulnerability reporters in the past). I tried to use those GitHub actions some years ago, and disabled them long time ago. So just removed those files.

**Comment by asi-cider** [comment]
@odinserj, thanks. Is it ok with you that I publicly disclose this case to raise awareness of this security issue?

**Comment by odinserj** [comment]
Yes sure, it was my fault I didn't answer you privately – sorry for this. It didn't cause any troubles, since GitHub actions aren't used in this repository. Thank you for the hint anyway!

**Comment by asi-cider** [comment]
Thank you!
